apps/discover.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/ash
+
+# shellcheck shell=dash
+
+arch=$(uname -m)
+packet_base_url=$(sed -nr 's|.*\bpacket_base_url=(\S+).*|\1|p' /proc/cmdline)
+facility=$(sed -nr 's|.*\bfacility=(\S+).*|\1|p' /proc/cmdline)
+tinkerbell=$(sed -nr 's|.*\btinkerbell=(\S+).*|\1|p' /proc/cmdline | sed -e 's|^http://||' -e 's|/.*||')
+phone_home_url="http://$tinkerbell/phone-home"
+
+discover_url=$(sed -nr 's|.*\bdiscover_url=(\S+).*|\1|p' /proc/cmdline)
+if [ -z "$discover_url" ]; then
+	discover_url="$tinkerbell/discover-os"
```

Comment:
It appears $tinkerbell is just the address, so we'll need to prepend `http://` here

Reply:
Fixed.